frontend/src/index.tsx

```diff
@@ -119,26 +120,28 @@ export const ReactApp = (
   <ApolloProvider client={client}>
     <React.StrictMode>
       <Provider store={store}>
-        <Router basename={process.env.PUBLIC_URL}>
-          <TelemetryProvider>
-            <PrimeErrorBoundary>
-              <Routes>
-                <Route path="/health/*" element={<HealthChecks />} />
-                <Route path="/pxp/*" element={<PatientApp />} />
-                <Route path="/uac/*" element={<AccountCreationApp />} />
-                <Route path="/sign-up/*" element={<SignUpApp />} />
-                <Route
-                  path="/register/:registrationLink"
-                  element={<SelfRegistration />}
-                />
-                <Route path="/session-timeout" element={<SessionTimeout />} />
-                <Route path="/reload-app" element={<Navigate to="/" />} />
-                <Route path="/*" element={<App />} />
-                <Route element={<>Page not found</>} />
-              </Routes>
-            </PrimeErrorBoundary>
-          </TelemetryProvider>
-        </Router>
+        <WithFeatureFlags>
+          <Router basename={process.env.PUBLIC_URL}>
+            <TelemetryProvider>
```

Comment:
Before this merges, we should double-check that it doesn't interfere with Azure's Telementry reporting at all...it shouldn't, but I don't really trust Azure to do the sensible thing anymore

Reply:
How can that be tested 🤔 ?

Reply:
We'll need to deploy to a testing environment and then check that metrics are still coming in through for frontend events. I'm happy to pair with you on doing that!

Reply:
Can someone confirm once this is complete? I'd really appreciate it!

Reply:
@johanna-skylight As I'm reviewing your PR, wanted to follow up on this ask. Please let me know if I can help check since I have Azure access now 😸

Reply:
Just followed up on this and the telemetry appears unaffected - events are still being logged in Azure. I think this PR is good to go from a metrics perspective.

Reply:
Thank you for checking! That was the only thing for me and now the merge conflict I introduced when I merged in my PR a couple days ago 😅  -- otherwise this looks ready to go to me too @johanna-skylight